ec2-spot-emr/emr-spot/run_example.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+mvn exec:java -Dexec.mainClass="aws.example.emr.CreateEmrFleet" -Dexec.cleanupDaemonThreads=false
```

Comment:
Perhaps we should decouple this, create a shaded/uber jar,  that takes a few parameters, and build it up + distribute, execute within the same container. 

The current approach has a lot of wiring. Whomever want's to understand what's is being executed has to see first that we are using, docker just to provide the maven and JDK execution environment, then docker run + environments to run a bash with environment variables, and then maven to compile and execute a hardcoded example.  If there is anything we can do to drive the attention to the best practices and the java example including a bit of simplification and parametrisation, I'm sure that will be welcome.